rules/linux/lnx_file_deletion.yml

```diff
@@ -0,0 +1,23 @@
+title: File Deletion
+id: 30aed7b6-d2c1-4eaf-9382-b6bc43e50c57
+status: stable
+description: Detects file deletion commands
+author: Ömer Günal, oscd.community
+date: 2020/10/07
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1070.004/T1070.004.md
+logsource:
+    product: linux
+    category: process_creation
+detection:
+    selection:
+        Image|endswith:
```

Comment:
At least rm is very usual on Unix systems and causes a lot of false positives. I don't see the value for generating detections from rm usage.

Reply:
Hello @thomaspatzke! 
We've discussed this topic before (in slack) and also here in GitHub, in an [issue](https://github.com/SigmaHQ/sigma/pull/1249#issuecomment-754862329):

>> @thomaspatzke: I expect a huge amount of FPs
>
> @yugoslavskiy: You are right. There is no way to differentiate the legitimate execution of find/ls from malicious use. So it is a behavior detection with low confidence. As there is no such metric in Sigma (at least, for now), I've used `level: low`. It's more like an event tagged with ATT&CK technique, rather than an alert, so it supposed to be used for enrichment and farther severity calculation, hunting, and investigations. Basically, it is an approach that was used for many other rules in the repo with `level: low`

I will copy the link to this thread in the other similar comment of yours so we will keep the discussion here.

Please let me know how you would like to proceed.

Reply:
Now I remember 😄 I had the detection use-case in mind, not the event enrichment while I was reviewing.

Florian and I discussed this and to make it possible to distinguish both use cases clearly, we've decided to add another level "informational" which is intended for this purpose and explicitely not for detection/alerting. I will then re-level these rules.